react-to-do/src/App.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import './App.css';
+
+function Todo({ todo, index, completeTodo, removeTodo }) {
+  return (
+    <div
+      className="todo"
+      style={{ textDecoration: todo.isCompleted ? "line-through" : "" }}
+    >
+      {todo.text}
+      <div>
+        <button onClick={() => completeTodo(index)}>Complete</button>
+        <button onClick={() => removeTodo(index)}>x</button>
+      </div>
+    </div>
+  );
+};
+
+function TodoForm({ addTodo }) {
+  const [value, setValue] = React.useState("");
+
+  const handleSubmit = e => {
```

Comment:
You may be able to do 
```suggestion
    const handleSubmit = (e, {value}) => { stuff };
```
Although I'm not certain, you'll have to check if a second variable is available, but I'm fairly certain it is. If not a common one is e.target.value also. I see you did e.target.value below, I typically do it in the handler, its half a dozen one or another really - sometimes you want to pull out more event stuff than the single value.

I actually suspect you have further errors due to passing e.target.value, pass e, then e.preventDefault() etc can be done, and is more intuitive....